scripts/check_for_inconsistent_pandas_namespace.py

```diff
@@ -17,28 +17,17 @@
 PATTERN = r"""
     (
         (?<!pd\.)(?<!\w)    # check class_name start with pd. or character
-        {class_name}\(      # match DataFrame but not pd.DataFrame or tm.makeDataFrame
+        ([A-Z]\w+)\(      # match DataFrame but not pd.DataFrame or tm.makeDataFrame
         .*                  # match anything
-        pd\.{class_name}\(  # only match e.g. pd.DataFrame
+        pd\.\2\(  # only match e.g. pd.DataFrame
     )|
     (
-        pd\.{class_name}\(  # only match e.g. pd.DataFrame
+        pd\.([A-Z]\w+)\(  # only match e.g. pd.DataFrame
         .*                  # match anything
         (?<!pd\.)(?<!\w)    # check class_name start with pd. or character
-        {class_name}\(      # match DataFrame but not pd.DataFrame or tm.makeDataFrame
+        \4\(      # match DataFrame but not pd.DataFrame or tm.makeDataFrame
```

Comment:
also, when you do the rebase, can you move the comments forwards so they all aligned again?